server/test/question.test.js

```diff
@@ -0,0 +1,211 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+
+import app from "../index.js";
+import { queryPromise } from "../db/database.js";
+import { forceTruncateTables, insertData } from "./index.test.js";
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe("Question generation with empty database", () => {
+  before("Remove all data", (done) => {
+    forceTruncateTables("molecule", "class", "system", "property_value", "molecule_property", "property").then(() =>
+      done()
+    );
+  });
+
+  for (let type = 1; type <= 10; ++type) {
+    it("Question type " + type, async () => {
+      const res = await requestAPI("question/" + type);
+      expect(res.status).equals(422);
+    });
+  }
+});
+
+describe("Question generation", function () {
+  const questionTypes = [1, 2, 3, 5, 6, 7, 8, 9];
```

Comment:
Why not also `10`?

Reply:
With the current data we can't generate question of type 4 and 10.

Reply:
Ok, I think a comment could be useful to clarify this 😉